core/src/main/java/com/graphhopper/routing/ch/CHParameters.java

```diff
@@ -0,0 +1,17 @@
+package com.graphhopper.routing.ch;
+
+import com.graphhopper.util.Parameters;
+
+public final class CHParameters {
+    static final String PERIODIC_UPDATES = Parameters.CH.PREPARE + "updates.periodic";
```

Comment:
I kind of see the point of reusing `Parameters.CH.PREPARE` here but I would prefer inlining these constants here (e.g. `"prepare.ch.updates.periodic"` instead of `Parameters.CH.PREPARE + "updates.periodic"`, because this makes it easier to search for this setting in the code base (I imagine seeing this setting in some configuration file and the first thing I would do would be to do a full text search to find out what it does in the code). But maybe this is for another discussion and have left this as it was.

Reply:
Yeah, we have this for the Parameter class too. Not sure about it, because now you can click "find usage" and no need for text search ;)

Reply:
I was thinking about the case that you only know the name of the parameter (from a config file for example) in which case you would do a text search (you cannot search for usages at this point because you do not know that this constant exists).  I think when the parameter value is half constant and half string you might miss it when you search for the full parameter name.

Reply:
of course doing it this way allows finding everything related to PREPARE.CH, but not sure if this is needed that often ?

Reply:
Yes. Probably this would be more consistent. In earlier version I had no such constants (like there are still a few), but this is also suboptimal.